samples/helpers/get-env/inc/get_env.hpp

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// SPDX-License-Identifier: MIT
+// Licensed under the MIT License.
```

Comment:
Since we are fixing these to be consistent with the guidance, shall we update line 1 as well to remove `All rights reserved.` and match it exactly?
https://docs.opensource.microsoft.com/releasing/general-guidance/copyright-headers/#c-1
```text
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```